extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/GrpcUtil.java

```diff
@@ -17,7 +17,7 @@ public class GrpcUtil {
     private static final Logger log = LoggerFactory.getLogger(GrpcUtil.class);
 
     public static StatusRuntimeException securelyWrapError(final Logger log, final Throwable err) {
-        return securelyWrapError(log, err, Code.INVALID_ARGUMENT);
+        return securelyWrapError(log, err, Code.ABORTED);
```

Comment:
Cristian, Colin, and I dug in a bit and agree that ABORTED is an acceptable alternative. Per our discussion, Ryan, it would be better to change this in isolation swapping the status type for other similar cases.